tests/suites/test_suite_bignum_mod_raw.function

```diff
@@ -287,6 +287,97 @@ exit:
 
 /* BEGIN MERGE SLOT 5 */
 
+/* BEGIN_CASE */
+void mpi_mod_raw_add( char * input_A, char * input_B,
+                      char * input_N, char * input_S )
```

Comment:
Changing the order of the input parameters will simplify the python arguments override

Reply:
This seems to be a natural order for these parameters, A, B, N (being the order they are passed to the function under test `mbedtls_mpi_mod_raw_add()` followed by the expected sum `S`.